Tests/XcodeGenKitTests/ProjectGeneratorTests.swift

```diff
@@ -274,10 +274,15 @@ class ProjectGeneratorTests: XCTestCase {
                 //     - framework: FrameworkA.framework
                 //     - framework: FrameworkB.framework
                 //       embed: false
+                // iOSFrameworkZ:
+                //   dependencies: []
```

Comment:
What's missing for this to be uncommented?

Reply:
This whole comment is explaining the targets below. I could remove the whole block, it just made it easier to reason about all of the dependencies.

Reply:
Yeah ok. You can also generate a spec from a yaml string if that's easier to declare 👍